components/event-handler-notification/org.wso2.carbon.identity.event.handler.notification/src/main/java/org/wso2/carbon/identity/event/handler/notification/util/NotificationUtil.java

```diff
@@ -149,6 +163,44 @@ public static Map<String, String> getPlaceholderValues(EmailTemplate emailTempla
 
         Map<String, String> configFilePlaceholders = getConfigFilePlaceholders();
 
+        JsonNode brandingPreferences = null;
+        Map<String, String> brandingFallbacks = getBrandingFallbacksFromConfigFile();
+
+        if (Boolean.parseBoolean(
+                IdentityUtil.getProperty(NotificationConstants.EmailNotification.ENABLE_ORGANIZATION_LEVEL_EMAIL_BRANDING))) {
+            try {
+                BrandingPreferenceManager brandingPreferenceManager = new BrandingPreferenceManagerImpl();
+                BrandingPreference responseDTO = brandingPreferenceManager.getBrandingPreference(
+                        BrandingPreferenceMgtConstants.ORGANIZATION_TYPE,
+                        placeHolderData.get("tenant-domain"),
```

Comment:
Can we use a constant for "tenant-domain" since it is used in more than one places?

Reply:
Fixed with 6554031cb7fa29639a7a02228afea911d08004a4